regions/io/read_ds9.py

```diff
@@ -192,20 +194,33 @@ def ds9_region_list_to_objects(region_list):
             else:
                 raise ValueError("No central coordinate")
         else:
+            # Note: this should effectively never happen, because it would
```

Comment:
I think this will happen all the time. People have invalid `ds9` files and `region_type`. They mistype or whatever.

Can you change this code comment from "this should effectively never happen, ..." to something like "For invalid inputs, we do ..."

Is it possible to give the user a line number where the problem in their file occurs?
It will be really common and users will appreciate a helpful message.